run_analysis.sh

```diff
@@ -40,6 +41,7 @@ function print_package_details {
 
 function print_results_dirs {
 	echo "Results dir: $RESULTS_DIR"
+	echo "File Write Results dir: $FILE_WRITE_RESULTS_DIR"
```

Comment:
Nit: can we keep the second column aligned when printed? (it's also aligned with package details)

Reply:
I resolved this manually, but let me know if there's a better way to do it.